src/flutter/shell/platform/linux_embedded/window/linuxes_window_drm.cc

```diff
@@ -13,7 +13,8 @@
 
 namespace flutter {
 
-static constexpr char kDrmDeviceFilename[] = "/dev/dri/card0";
+static constexpr char kFlutterDrmDevice[] = "FLUTTER_DRM_DEVICE";
```

Comment:
kFlutterDrmDevice -> kFlutterDrmDeviceEnvironmentKey

Reply:
I fixed it.